ccmlib/node.py

```diff
@@ -1043,30 +1022,33 @@ def run_sstablerepairedset(self, set_repaired=True, datafiles=None, keyspace=Non
         sstablerepairedset = common.join_bin(cdir, os.path.join('tools', 'bin'), 'sstablerepairedset')
         env = self.get_env()
         sstablefiles = self.__gather_sstables(datafiles, keyspace, column_families)
+        results = []
 
         for sstable in sstablefiles:
```

Comment:
Don't block merge on this, but it's worth checking to see if `sstablerepairedset` can take multiple tables. If it can, it'd be nice to only shell out once.